_component_design/help-beacon.md

```diff
@@ -18,44 +17,110 @@ helix-ui-javascript: false
 
 <section class="static-section" markdown="1">
 
-## Introduction
```

Comment:
It seems like you might have left some kind of introduction so you don't jump immediately into when to use a beacon without describing what a beacon is. I like the summary blurb in the metadata. Unless that blurb displays in the published book, I would add a similar sentence before the "When to use" section.